src/pydvl/influence/frameworks/util.py

```diff
@@ -0,0 +1,158 @@
+import copy
+import logging
+import math
+from typing import Dict, Iterable, Tuple, TypeVar, Union
+
+import torch
+
+logger = logging.getLogger(__name__)
+
+
+def to_model_device(x: torch.Tensor, model: torch.nn.Module) -> torch.Tensor:
+    """
+    Returns the tensor `x` moved to the device of the `model`, if device of model is set
+    :param x:
+    :param model:
+    :return:
```

Comment:
I don't think it's necessary to document the arguments for this function.

```suggestion
    Returns the tensor `x` moved to the device of the `model`, if device of model is set
    
```

Reply:
Done